.eslintrc.json

```diff
@@ -1,24 +0,0 @@
-{
-    "env": {
-        "browser": true,
-        "es2021": true
-    },
-    "parser": "babel-eslint",
-    "parserOptions": {
-        "ecmaVersion": 12,
-        "sourceType": "module"
-    },
-    "plugins": ["svelte3"],
-    "extends": ["eslint:recommended"],
-    "overrides": [
-        {
-        "files": ["**/*.svelte"],
-        "processor": "svelte3/svelte3"
-        }
-    ],
-    "rules": {
-        "quotes": ["error", "single"],
-        "indent": ["error", 2],
-        "semi": ["error", "always"]
-    }
-}
```

Comment:
same comment for editorconfig. Is there a reason to delete this file?

Reply:
I totally don't understand this file but seems that it is according to `sevelt` . This was causing a collision with `reactjs`.  `reactjs` comes configured with eslint. Can I write an .eslintrc according to `reactjs`?

Reply:
some properties are related to Svelte, but for example the `rules` aren't. You can write an .eslintrc for React, but with the same rules so that we keep them the same across branches 👍